drivers/rtt_rtc/Kconfig

```diff
@@ -11,5 +11,6 @@ config MODULE_RTT_RTC
     depends on TEST_KCONFIG
     select MODULE_PERIPH_RTT
     select MODULE_RTC_UTILS
+    imply MODULE_ZTIMER_NO_PERIPH_RTT
```

Comment:
I think you would have to model the different `rtt` users as a choice and let only one of them be active at the same time to do something like this.

Otherwise, select the module at the application level, and that is actually the right way to do so, see comment in 7c3360e4d6f988bc9d5b2cc72b6d6c20af377cf0, although I have violated this rule myself.

So either the dependency modeling needs to be improved, where the "owner" of the `periph_rtt` is modeled, or simply add the `ztimer_no_periph_rtt` to your application.